test_requirements.txt

```diff
@@ -5,6 +5,7 @@ behave==1.2.6
 biopython~=1.73
 black==v19.10.b0
 dask[complete]~=2.6
+datatable>=0.11.1, <1.0
```

Comment:
Datatable should also be added into `extras_require` in `setup.py`

Reply:
Good catch! Thanks.

I'm not really familiar with the `extras_require` configuration for setuptools. I have added requirements as far as I get the logic, but could you please have a look that it is correct? In particular the cross-dependency of entries with datatable and pandas requirements is not fully clear to me.